eng/Baseline.xml

```diff
@@ -10,6 +10,7 @@ build of ASP.NET Core 2.2.x. Update this list when preparing for a new patch.
   <Package Id="dotnet-user-secrets" Version="2.2.0" />
   <Package Id="dotnet-watch" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.Antiforgery" Version="2.2.0" />
+  <Package Id="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" Version="2.2.0-preview-35687" />
```

Comment:
This never shipped RTM. I'm told it will ship at some point with another 2.2.x release, and that it should ship with a 2.2.0 baseline.

https://github.com/aspnet/AspNetCore/blob/7e38b563bdbfe240bcce52e76329c70e45110df6/src/Identity/src/ApiAuth.IS/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.csproj#L4-L5